automation/taskcluster/androidTest/ui-test.sh

```diff
@@ -84,8 +84,15 @@ samples=${component//samples-}
 if [[ "${component}" != samples-* ]]
 then
     # Case 1: tests for any component (but NOT samples, NOT real UI tests)
-    APK_APP="./samples/${component}/build/outputs/apk/geckoNightly/debug/samples-${component}-geckoNightly-debug.apk"
```

Comment:
I'm not sure why we need to pass these to parameter to FLANK when we just want to run instrumentation test no UI test.

Reply:
Was not sure there could be a different sample that could run similar tests. If there is  not any and it is not going to be, then, no need to parameter that

Reply:
Thanks 👍

Reply:
It looks like Flank needs the extra parameter without it the [compilation fails](https://firefoxci.taskcluster-artifacts.net/VZDSa4dBRTmzW9X29UI9fw/0/public/logs/live_backing.log). I will left the fixed string as it is. 

```
 /usr/bin/java -jar /builds/worker/test-tools/flank.jar android run --config=./automation/taskcluster/androidTest/flank-arm.yml --max-test-shards=1 --test=./components/feature/pwa/build/outputs/apk/androidTest/debug/feature-pwa-debug-androidTest.apk --project=moz-android-components-230120 exitcode=0
 version: v20.05.2
 revision: 7618ebdb5a74db4b448de0b93357221de3ab10c2
 
 Unmatched argument at index 6: 'exitcode=0'
 Run tests on Firebase Test Lab
```